lib/sidekiq/undertaker/web_extension/api_helpers.rb

```diff
@@ -112,13 +127,40 @@ def view_path
 
         def redirect_path(request)
           path = request.referer ? URI.parse(request.referer).path : request.path
-          path.gsub("/delete", "").gsub("/retry", "")
+          path.gsub("/delete", "").gsub("/retry", "").gsub("/export", "")
+        end
+
+        def prepare_data(data, chunk_size)
+          filename = Time.now.strftime("%Y-%m-%d_%H-%M").to_s
```

Comment:
Shouldn't `req_job_class` already be part of `filename` here, so that it also added in case of `data.length <= chunk_size`?

```ruby
filename = "#{@req_job_class}_#{Time.now.strftime("%Y-%m-%d_%H-%M")"
```

Reply:
Yes and no. If we are using the button ExportAll, then we have the variable `@req_job_class`, but if we are manually selecting jobs to export, then we never get that data on the request